plugins/graph.js

```diff
@@ -204,21 +204,21 @@ function reset_agg () {
 }
 
 exports.get_data = function (res, earliest, today, group_by) {
-    const next_stop = earliest + group_by;
+    const next_stop = `${earliest}${group_by}`;
     const aggregate = reset_agg();
     const plugin = this;
 
     function write_to (data) {
         // plugin.loginfo(data);
-        res.write(data + "\n");
+        res.write(`${data} \n`);
```

Comment:
You've inadvertently added a whitespace char here.

Reply:
fixed.